shipyard_blueprints/dropbox/pyproject.toml

```diff
@@ -5,18 +5,20 @@ description = "A local client to connect and work with Dropox"
 authors = ["wrp801 <wespoulsen@gmail.com>"]
 license = "Apache 2.0"
 readme = "README.md"
-packages = [{include = "shipyard_dropbox"}]
+packages = [{ include = "shipyard_dropbox" }]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.10"
-dropbox = "11.36.0"
-shipyard-templates = "0.5.0a0"
+python = "^3.9"
+shipyard-templates = "^0.7.0"
```

Comment:
There will exist a later version of this (0.8.0), not that it affects this but just so you know